package.json

```diff
@@ -71,6 +72,9 @@
         "typescript": "5.2.2",
         "vitest": "^0.34.5"
     },
+    "optionalDependencies": {
```

Comment:
This isn't quite what we want.  `optionalDependencies` will always be attempted to be installed, and the only difference is that `npm install` will not fail if the package fails to install (due to being built for a different computer architecture, for instance).  

Instead, we can move this to `peerDependencies`, and add an `optional: true` in `peerDependenciesMeta`, similar to what we have for `@vue/compiler-sfc`.